tsconfig.json

```diff
@@ -1,22 +1,14 @@
 {
   "compilerOptions": {
     "moduleResolution": "node",
-    "target": "es6",
-    "module":"es2015",
-    "lib": ["es2016", "es2017", "dom"],
+    "target": "es5",
```

Comment:
I think we can use es6 spec, or not?

Reply:
If we will use es6 then projects which use this lib will need to use typescript or babel loader for legacy browsers, I think for now is better to stay with es5, is simple to change in future.

Reply:
`storeon` itself use [es6 spec](https://github.com/storeon/storeon/commit/82cecf51c9fd38d1e62875dc4df5087899b2743a), so user anyway needs to use typescript or babel loader for legacy browsers

Reply:
You right, ok lets go to es6.